experimental/PropertyDDS/packages/property-properties/src/properties/baseProperty.ts

```diff
@@ -670,7 +666,7 @@ export abstract class BaseProperty {
 	 *
 	 * @returns the flat representation
 	 */
-	private _flatten(): object {
+	protected _flatten(): object {
 		return { propertyNode: this };
```

Comment:
TypeScript 5 noticed a private method was colliding with another private method.

In this case, I think the intention was to override, and so I promoted '_flatten' to protected in both cases.